twemproxy/README.md

```diff
@@ -26,6 +26,15 @@ The Agent's Twemproxy check is included in the [Datadog Agent][2] package, so yo
 
 2. [Restart the Agent][5] to begin sending twemproxy metrics to Datadog.
 
+#### Containerized
+For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying the parameters below.
+
+| Parameter            | Value                                  |
+|----------------------|----------------------------------------|
+| `<INTEGRATION_NAME>` | `twemproxy`                            |
+| `<INIT_CONFIG>`      | blank or `{}`                          |
+| `<INSTANCE_CONFIG>`  | `{"host": "%%host%%", "port":"22222"}` |
```

Comment:
seems there's an extra `2` compared to `port: 2222`

Reply:
```suggestion
| `<INSTANCE_CONFIG>`  | `{"host": "%%host%%", "port":"2222"}` |
```